### ISSUE TITLE
Module missing? Everything broken? Ahhh!

### ISSUE THREAD

**Issue by bixel** [Kecksdose/shiftsum#1]
The app reports `NameError: name 'datetime' is not defined` when running with some downloaded csv-logbook:
```
shiftsum -r ~/Downloads/rundbfills.csv
Traceback (most recent call last):
  File "/usr/local/bin/shiftsum", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/shiftsummary/command_line.py", line 38, in main
    shiftsummary(args.rundb, args.shifters, args.output_dir)
  File "/usr/local/lib/python3.6/site-packages/shiftsummary/__init__.py", line 114, in shiftsummary
    datetime.timedelta(seconds=duration) for duration in df["time_total"]
  File "/usr/local/lib/python3.6/site-packages/shiftsummary/__init__.py", line 114, in <listcomp>
    datetime.timedelta(seconds=duration) for duration in df["time_total"]
NameError: name 'datetime' is not defined
```

**Comment by Kecksdose** [comment]
Should be fixed by b08b9a34c. Could you try to update via pipy?

**Comment by Kecksdose** [comment]
Okay, finally fixed in b91c4438f8a59cfa1 :)
